src/routes/write-exercise/components/InteractionSection.tsx

```diff
@@ -186,9 +186,13 @@ const InteractionSection = (props: InteractionSectionProps): ReactElement => {
       )}
       {retryAllowed && (
         <Pressable onPress={Keyboard.dismiss}>
-          <Button onPress={checkEntry} disabled={!input} buttonTheme={BUTTONS_THEME.dark} testID='check-entry'>
-            <LightLabelInput disabled={!input}>{labels.exercises.write.checkInput}</LightLabelInput>
```

Comment:
Can `LightLabelInput` styled component above be removed?

Reply:
yes, good catch.